packages/block-library/src/embed/embed-placeholder.js

```diff
@@ -31,15 +32,17 @@ const EmbedPlaceholder = ( {
 			) }
 		>
 			<form onSubmit={ onSubmit }>
-				<input
+				<InputControl
+					__next40pxDefaultSize
 					type="url"
 					value={ value || '' }
-					className="components-placeholder__input"
-					aria-label={ label }
+					className="wp-block-embed__placeholder-input"
+					label={ __( 'Embed URL' ) }
```

Comment:
Should this be `__( 'Embed URL' )` or should it just be `label`?

Reply:
Good catch! I shouldn't have used a hard-coded string here. We should reference [this label](label) and display the corresponding label for the variation. I'll merge it after I update.